resources/state_of_the_dapps.json

```diff
@@ -0,0 +1,43 @@
+{
+    "auction-house":
+    {
+        "name":
+        {
+            "en": "Auction House"
+        },
+        "photo-path": "contacts://auction-house",
+        "state-of-the-dapp?": true,
```

Comment:
@viebel I'll try to explain how it should work. Because we want to edit/delete default groups (DApps and State of the DApps) we should load it from json only once on account creation, and store it all in realm as contacts and create and store two groups, "DApps" and "State of the DApps", because user can delete default contacts and DApps and they shouldn't appear when user login next time. And also user will create new groups which will be stored in realm and there shouldn't be any difference between default groups and created by the user. There will be one function get-all-groups and it should return all groups from realm, defaults (created from json on account creation) and created by user. I'm currently working on group object's structure, and can't exactly say how it should look like, something like
```{:group-id     "dapps"
       :name        "DApps"
       :timestamp   (.getTime (js/Date.))
       :contacts    contacts}```

So, we need
1)dispatch :load-default-contacts! only once on account creation
   -leave contacts creation unchanged (where is no difference between regular dapps and state-of-the-dapps, they all are dapps?=true)
   -create and save two new groups "DApps" and "State of the DApps", here we need info about difference between regular dapps and state-of-the-dapps. But i should implement functions for creation and saving groups first.
2) remove 
```dapps                    (subscribe [:get-added-dapps-with-limit contacts-limit])
     state-of-the-dapps       (subscribe [:get-added-state-of-the-dapps-with-limit contacts-limit])
      dapps-count              (subscribe [:added-dapps-count])
      state-of-the-dapps-count (subscribe [:added-state-of-the-dapps-count])
 ```
3) implement ```groups  (subscribe [:get-all-groups])```